src/views/Pools/components/BountyModal.tsx

```diff
@@ -75,9 +80,18 @@ const BountyModal: React.FC<BountyModalProps> = ({
       <Flex alignItems="flex-start" justifyContent="space-between">
         <Text>{t("You'll claim")}</Text>
         <Flex flexDirection="column">
-          <Text bold>{cakeBountyToDisplay} CAKE</Text>
+          <Balance bold value={cakeBountyToDisplay} decimals={7} unit=" CAKE" />
```

Comment:
Is it just for debugging and you are planing to bring it back to 3 decimals (as now in prod)? Also for some reason its not even displaying decimals. And even if it would it probably won't fit in there 😄